style.css

```
@@ -0,0 +1,113 @@
+
+/*General Page Style */
+*{
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: sans-serif;
+}
+
+/*Profile Section at Home Page Style */
+.main_container{
+	height: 460px;
+	background-image: url(real-estate-agent-with-house-model-keys.jpg);
+	background-position: center;
+	background-size:cover;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+#main_container{
+	padding: 6%;
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.main_container_header{
+	padding-bottom: 20px;
+	color: #0f4b5e;
+	font-size: 48px;
+}
+
+
+
+.main_header_content{
+	color: #071144;
+}
+
+.main_container_list{
+	list-style: none;
+	padding-bottom: 20px;
+	padding-top: 20px;
+	padding-left: 120px;
+	display: flex;
+	text-align: center;
+
+	
+}
+
+.main_container_list_botton{
+	border: none;
+	background: unset;
+	font-size: 16px;
+	padding-left: 20px;
+	
+
+}
```

here, it's better to update the style as the following : 

* use short syntax for `padding`
* to center the 2-buttons you can use just `justify-content : space-around` and `margin : 0 auto ` then delete  `text-align` & `padding-left` that have no-effect. 
* make the width of  the list : `50%`
* delete `padding-left : 20px` from the class `.main_container_list_botton`
```suggestion
.main_container_list{
	.main_container_list{
    list-style: none;
    padding: 20px 0px;
    display: flex;
    justify-content: space-around;	
    width : 50%
}

.main_container_list_botton{
	border: none;
	background: unset;
	font-size: 16px;	
}
```